### PR TITLE
css and buttons

### PR DESCRIPTION
**Pull request by Polkas**
linked to https://github.com/insightsengineering/coredev-tasks/issues/225
linked to https://github.com/insightsengineering/teal.reporter/issues/88

- CSS review
- buttons style;  icons with dynamic labels
- removed not used failed arg in modals

buttons style:

https://user-images.githubusercontent.com/10676545/180709996-f508e9c4-3dbd-4822-947d-e25425e4a026.mov

### CODE REVIEW

File: R/AddCardModule.R [hunk 1]
```diff
@@ -4,9 +4,10 @@
 #'
 #' For more details see the vignette: `vignette("simpleReporter", "teal.reporter")`.
 #' @param id `character(1)` this `shiny` module's id.
+#' @param label `character(1)` label before the icon.
 #' @return `shiny::tagList`
 #' @export
-add_card_button_ui <- function(id) {
+add_card_button_ui <- function(id, label = "") {
```
Comment (mhallal1) [inline R/AddCardModule.R:10]: lets remove the label here and give a static label when hovering over the button.

File: R/DownloadModule.R [hunk 2]
```diff
@@ -4,18 +4,22 @@
 #'
 #' For more details see the vignette: `vignette("simpleReporter", "teal.reporter")`.
 #' @param id `character(1)` this `shiny` module's id.
+#' @param label `character(1)` label before the icon.
 #' @return `shiny::tagList`
 #' @export
-download_report_button_ui <- function(id) {
+download_report_button_ui <- function(id, label = "") {
```
Comment (mhallal1) [inline R/DownloadModule.R:10]: same here

**Comment by mhallal1** [comment]
Warnings:
Screenshot from 2022-07-22 15-07-16

File: R/AddCardModule.R [hunk 3]
```diff
@@ -4,18 +4,28 @@
 #'
 #' For more details see the vignette: `vignette("simpleReporter", "teal.reporter")`.
 #' @param id `character(1)` this `shiny` module's id.
+#' @param label `character(1)` label before icon, if used then dynamic hover label is not available.
+#' By default `NULL` so a dynamic hover label is used.
 #' @return `shiny::tagList`
 #' @export
-add_card_button_ui <- function(id) {
+add_card_button_ui <- function(id, label = NULL) {
```
Comment (mhallal1) [inline R/AddCardModule.R:11]: no need for `label`, this will be standard.

Comment (Polkas) [reply R/AddCardModule.R]: done

**Review by mhallal1 — APPROVED** [review]

**Pull request merged**